src/components/Dashboard.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div> 
+    <table  align="center">
```

Comment:
スペースが半角2つ分になっているようです。
```suggestion
    <table align="center">
```

Reply:
スペース消しました。